src/Core/Components/List/FluentPersona.razor.cs

```diff
@@ -49,6 +49,12 @@ public partial class FluentPersona : FluentComponentBase
     [Parameter]
     public string? ImageSize { get; set; }
 
+    /// <summary>
+    /// Gets or sets the position of the image.
+    /// </summary>
+    [Parameter]
+    public PersonaImagePosition? ImagePosition { get; set; }
```

Comment:
I wouldn't put a nullable property, but with a default value of "Right/After".

Reply:
End 😉